docs/contributing/plugin.md

```
@@ -805,6 +805,253 @@ func (p *AsyncPlugin) PostHook(
+       return resp
+   }
+   ```
+
+5. **Documentation:**
+   - Document streaming behavior in plugin README
+   - Explain state management approach
+   - List supported delta types
+   - Provide streaming-specific examples
+
+> **💡 Tip**: Use the `isStreamingResponse` helper to reliably detect streaming responses:
+
```

markdownlint-cli2 (0.17.2)

Blank line inside blockquote

(MD028, no-blanks-blockquote)